### ISSUE TITLE
[Serious] State: Expand/collapse state of the element is missing or incorrect

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#641]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87816 |
| **Checkpoint** | 4.1 Compatible › Name, Role, Value (4.1.2.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 05 Create New Workflow (Workflow) |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/12/26 at 6:22 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The expand/collapse state of a toggle-type element is missing or is used incorrectly.
At the end of New Job workflow, if you select Create Workflow for a project with the same name, an Error dialog appears, there is a Show Details button that expands but there is no way to collapse.  See Create Job User Experience video at 31:00.

## Recommendation

Rule : 
The name, role, value, states, and properties of user interface components MUST be programmatically determinable by assistive technologies.

How To Fix: 
Fix this issue by using the aria-expanded attribute on the actionable toggle element (e.g. the <button> element) with a value of "true" (when the section is expanded) or "false" (when the section is collapsed).

<h3><button aria-expanded="true">When will I receive my refund?</button></h3>
<div class="faq-expanded">You should receive your refund within 5 business days....</div>

Reference: 
W3C ARIA Recommendation: https://www.w3.org/TR/wai-aria-1.1/#aria-expanded

Background: 
When collapsible content is used, a screen reader user must be able to determine whether the collapsible content below the toggle element is in the expanded state or in the collapsed state.

## Screenshots

![05_no collapse](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87816/05_no_collapse.png)
